.github/workflows/publish.yaml

```diff
@@ -12,144 +12,51 @@ jobs:
             version: v0.9.14
-            repository: https://github.com/Sifchain/sifnode
-            namespace: SIFNODED
-            build_env: default
           - project: bitcanna
-            project_bin: bcnad
-            project_dir: .bcna
             version: v1.2
-            repository: https://github.com/BitCannaGlobal/bcna
-            namespace: BCNAD
-            build_env: default
           - project: emoney
-            project_bin: emd
-            project_dir: .emd
             version: v1.1.3
-            repository: https://github.com/e-money/em-ledger
-            namespace: EMD
-            build_env: default
           - project: bitsong
-            project_bin: bitsongd
-            project_dir: .bitsong
             version: v0.8.0
-            repository: https://github.com/bitsongofficial/go-bitsong
-            namespace: BITSONG
-            build_env: default
           - project: cryptoorgchain
-            project_bin: chain-maind
-            project_dir: .cryptoorgchain
             version: v3.1.1-croeseid
-            repository: https://github.com/crypto-org-chain/chain-main
-            namespace: CRYPTOORGCHAIN
-            build_env: default
           - project: impacthub
-            project_bin: ixod
-            project_dir: .ixod
             version: v1.6.0
-            repository: https://github.com/ixofoundation/ixo-blockchain
-            namespace: IXOD
-            build_env: default
           - project: band
-            project_bin: bandd
-            project_dir: .band
             version: v2.3.2
-            repository: https://github.com/bandprotocol/chain
-            namespace: BAND
-            build_env: default
           - project: kichain
-            project_bin: kid
-            project_dir: .kid
             version: 2.0.0
-            repository: https://github.com/KiFoundation/ki-tools
-            namespace: KICHAIN
-            build_env: default
+          - project: likecoin
+            version: fotan-1.1
     steps:
       - uses: actions/checkout@v2
       - name: build
         run: |
+          # Create dummy env file
+          touch .env
           echo "${{ secrets.GHCR_TOKEN }}" | docker login ghcr.io -u "${{ secrets.GHCR_USER }}" --password-stdin
-          docker build . --tag ghcr.io/${{ github.repository }}:${GITHUB_REF#refs/tags/}-${{matrix.project}}-${{matrix.version}}${{matrix.tag_suffix}} \
-            --build-arg PROJECT=${{matrix.project}} \
-            --build-arg PROJECT_BIN=${{matrix.project_bin}} \
-            --build-arg PROJECT_DIR=${{matrix.project_dir}} \
-            --build-arg VERSION=${{matrix.version}} \
-            --build-arg REPOSITORY=${{matrix.repository}} \
-            --build-arg NAMESPACE=${{matrix.namespace}} \
-            --build-arg BUILD_ENV=${{matrix.build_env}} \
-            --build-arg WASMVM_VERSION=${{matrix.wasmvm_version}}
+          cd ${{matrix.project}}
+          docker-compose build
```

Comment:
i like this. no need to duplicate build args. this way you can easily extend the current build args list.